lib/Wikia/src/Service/User/Attributes/UserAttributes.php

```diff
@@ -91,9 +97,51 @@ private function setInMemcache( $userId, $attributes ) {
 	 * @param Attribute $attribute
 	 */
 	public function setAttribute( $userId, Attribute $attribute ) {
-		$this->setInService( $userId, $attribute );
+		$this->loadAttributes( $userId );
```

Comment:
Are there any usages of `$this->attributes` in this method. Why is this call needed?